site/docs/pages/transaction/transaction.mdx

```diff
@@ -123,6 +123,63 @@ export default function TransactionComponents() {
   </TransactionWrapper>
 </App>
 
+## `Transaction` with Paymaster Gas Sponsorship Capabilities
+##### Obtain a Paymaster and Bundler endpoint from the [Coinbase Developer Platform Paymaster](https://portal.cdp.coinbase.com/products/bundler-and-paymaster).
+
+<img
+  alt="OnchainKit Paymaster and Bundler endpoint"
+  title="OnchainKit Paymaster and Bundler endpoint"
+  src="/assets/onchainkit-components-paymaster-endpoint.png"
```

Comment:
In the image we completelly cover the URL. I have a feeling people might not know that is an URL, maybe we should only cover the last part where the Key is exposed.

Reply:
updated